proto/zepben/protobuf/ns/data/change-status.proto

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2021 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+syntax = "proto3";
+option java_multiple_files = true;
+option java_package = "com.zepben.protobuf.nm";
+option csharp_namespace = "Zepben.Protobuf.NM";
+package zepben.protobuf.nm;
+
+import "google/protobuf/timestamp.proto";
+
+/**
+ * A response indicating all items in the batch were applied successfully.
+ */
+message BatchSuccessful {
+}
+
+/**
+ * A response indicating all items in the batch were applied successfully.
```

Comment:
Comment is wrong - assumedly idea here is that "if paused, tell client that we're paused, but they shouldn't retry"

Reply:
Yes this is a copy/paste error and has been fixed.